From 9f4c619c13dfe89635f3e88325e887b1cf82334e Mon Sep 17 00:00:00 2001
From: John Greene <jogreene@redhat.com>
Date: Thu, 14 Aug 2014 14:42:23 -0400
Subject: [ethernet] e1000e: remove unnecessary break after return

Message-id: <1408027348-9610-30-git-send-email-jogreene@redhat.com>
Patchwork-id: 87719
O-Subject: [RHEL7.1 PATCH 29/34] e1000e: remove unnecessary break after return
Bugzilla: 1091119
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

BZ: 1091119
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7834849
Tested: By me. Acceptable. See BZ for details

Signed-off-by: Fabian Frederick <fabf@skynet.be>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 4145ce0f59691820ea7e2c2be5d1c34aab61560e)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/e1000e/82571.c b/drivers/net/ethernet/intel/e1000e/82571.c
index 218481e..dc79ed8 100644
--- a/drivers/net/ethernet/intel/e1000e/82571.c
+++ b/drivers/net/ethernet/intel/e1000e/82571.c
@@ -95,7 +95,6 @@ static s32 e1000_init_phy_params_82571(struct e1000_hw *hw)
   break;
  default:
   return -E1000_ERR_PHY;
-  break;
  }
 
  /* This can only be done after all function pointers are setup. */
@@ -422,7 +421,6 @@ static s32 e1000_get_phy_id_82571(struct e1000_hw *hw)
   break;
  case e1000_82573:
   return e1000e_get_phy_id(hw);
-  break;
  case e1000_82574:
  case e1000_82583:
   ret_val = e1e_rphy(hw, MII_PHYSID1, &phy_id);
@@ -440,7 +438,6 @@ static s32 e1000_get_phy_id_82571(struct e1000_hw *hw)
   break;
  default:
   return -E1000_ERR_PHY;
-  break;
  }
 
  return 0;
@@ -1458,7 +1455,6 @@ static s32 e1000_setup_copper_link_82571(struct e1000_hw *hw)
   break;
  default:
   return -E1000_ERR_PHY;
-  break;
  }
 
  if (ret_val)
diff --git a/drivers/net/ethernet/intel/e1000e/ethtool.c b/drivers/net/ethernet/intel/e1000e/ethtool.c
index b04bad9..c0377d1 100644
--- a/drivers/net/ethernet/intel/e1000e/ethtool.c
+++ b/drivers/net/ethernet/intel/e1000e/ethtool.c
@@ -1519,11 +1519,9 @@ static int e1000_setup_loopback_test(struct e1000_adapter *adapter)
   switch (hw->mac.type) {
   case e1000_80003es2lan:
    return e1000_set_es2lan_mac_loopback(adapter);
-   break;
   case e1000_82571:
   case e1000_82572:
    return e1000_set_82571_fiber_loopback(adapter);
-   break;
   default:
    rctl = er32(RCTL);
    rctl |= E1000_RCTL_LBM_TCVR;
diff --git a/drivers/net/ethernet/intel/e1000e/ich8lan.c b/drivers/net/ethernet/intel/e1000e/ich8lan.c
index effbc26..da8f7bf 100644
--- a/drivers/net/ethernet/intel/e1000e/ich8lan.c
+++ b/drivers/net/ethernet/intel/e1000e/ich8lan.c
@@ -544,7 +544,6 @@ static s32 e1000_init_phy_params_ich8lan(struct e1000_hw *hw)
   break;
  default:
   return -E1000_ERR_PHY;
-  break;
  }
 
  return 0;
diff --git a/drivers/net/ethernet/intel/e1000e/mac.c b/drivers/net/ethernet/intel/e1000e/mac.c
index 8c386f3..30b74d5 100644
--- a/drivers/net/ethernet/intel/e1000e/mac.c
+++ b/drivers/net/ethernet/intel/e1000e/mac.c
@@ -787,7 +787,6 @@ static s32 e1000_commit_fc_settings_generic(struct e1000_hw *hw)
  default:
   e_dbg("Flow control param set incorrectly\n");
   return -E1000_ERR_CONFIG;
-  break;
  }
 
  ew32(TXCW, txcw);
-- 
1.7.1